worlds/aquaria/Regions.py

```diff
@@ -1070,6 +1070,10 @@ def __adjusting_manual_rules(self) -> None:
         add_rule(
             self.multiworld.get_location(AquariaLocationNames.THE_BODY_CENTER_AREA_BREAKING_LI_S_CAGE, self.player),
             lambda state: _has_tongue_cleared(state, self.player))
+        add_rule(self.multiworld.get_location(
+                    AquariaLocationNames.OPEN_WATERS_TOP_RIGHT_AREA_BULB_IN_THE_SMALL_PATH_BEFORE_MITHALAS,
+                    self.player), lambda state: _has_bind_song(state, self.player)
+        )
```

Comment:
```suggestion
        add_rule(self.get_location(
                    AquariaLocationNames.OPEN_WATERS_TOP_RIGHT_AREA_BULB_IN_THE_SMALL_PATH_BEFORE_MITHALAS),
                    lambda state: _has_bind_song(state, self.player)
        )
```
Just a tiny thing to use the shorter helper, though I guess that makes it look different from the ones surrounding it

Reply:
Ah. Did not know that. I will replace them, but later (as in the next PR) if it is correct for you.

Reply:
In fact, I just tried it and it does work. Here `self` is a Region, not  World.

Reply:
Ah, my bad, didn't realize. Also didn't realize that you could get `multiworld` off of `Region` (which I guess I should've known)
There's probably another way it could be done knowing that, but that'd be a pretty annoying refactor for almost no gain so eh